### DISCUSSION TITLE
Export collision data

### DISCUSSION THREAD

**Discussion by ytimber** [InteractiveComputerGraphics/SPlisHSPlasH#226] [Q&A]
Hi @janbender and all, I'm wondering whether it's possible to export collision data (i.e., when the liquid first collides with the surface of a solid object and how much liquid is on the surface). Thank you!

**Comment by janbender** [comment]
You will have to write an own exporter for this. Note there is no collision detection in SPH simulations but you can extract the info if a boundary is in the neighborhood of a fluid particle. You will get this information from the neighborhood search. 

Hope that helps!
